tests/business/emptyloop_test.py

```diff
@@ -58,7 +58,7 @@ async def test_run(
     }
 
     # Check events
-    published = cast(MockEventPublisher, events.empty_loop).published
+    published = cast("MockEventPublisher", events.empty_loop).published
```

Comment:
Why change this? I assume that the typechecker still knows the interface if we do, though I don't know how. And if we do, does it mean we should get rid of the `MockEventPublisher` import?

Reply:
This is just a Ruff thing. The latest version of Ruff always wants the first argument to `cast` to be quoted. See [TC006](https://docs.astral.sh/ruff/rules/runtime-cast-value/). I'm not sure I completely buy the logic, but it doesn't seem worth overriding since it doesn't really matter.